scripts/populate_placeholders.py

```diff
@@ -36,7 +36,7 @@ def main():
         "[[HYPERLINK_A029]]": "[A029 - Available actions for user list](#api-Default-a029-available-actions-for-user-list)",
         "[[HYPERLINK_A030]]": "[A030 - Retrieve e-RS business functions](#api-Default-a030-retrieve-business-functions)",
         "[[HYPERLINK_A033]]": "[A033 - Retrieve Healthcare Service](#api-Default-a033-retrieve-healthcare-service)",
-        "[[HYPERLINK_A034]]": "[A034 - Retrieve service version](#api-Default-a034-retrieve-service-version)",
+        "[[HYPERLINK_A034]]": "[A034 - Update Appointment](#api-Default-a034-update-appointment)",
```

Comment:
Looks like retrieve-service-version is also being replaced here, I think this should instead be being added as a separate entry for the new endpoint.

Reply:
Yes the retrieve-service-version is moved to A037and being developed in another branch,